app/main.py

```diff
@@ -293,10 +294,33 @@ async def data_not_exists_error_handler(request: Request, exc: DataNotExistsErro
     )
 
 
+def convert_errors(
+    e: ValidationError | RequestValidationError,
+) -> list[ErrorDetails]:
+    new_errors: list[ErrorDetails] = []
+    for error in e.errors():
+        # "url" field which Pydantic V2 adds when validation error occurs is not needed for API response.
+        # https://docs.pydantic.dev/2.1/errors/errors/
+        if "url" in error.keys():
+            error.pop("url", None)
```

Comment:
The URL values change with each Pydantic version upgrade, and test codes needs to be changed each time on Pydantic upgrade. To avoid this, included no URL attribute in error response validation even in unit test mode.
(If "url" attribute is required, I can fix this with reverting https://github.com/BoostryJP/ibet-Wallet-API/pull/1409/commits/09baf262e1c16599516f6d1ae92ba745c64a290d)